src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -225,6 +269,13 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
         }
     }
 
+    private void ValidateGVCFVariant(final VariantContext vc) {
```

Comment:
This should start with a lowercase letter, in java only types should start with a capital, ex classes and interfaces.